docs/contributing/contributing-rules.md

```diff
@@ -19,13 +19,17 @@ There are two ways in which you can contribute to the Semgrep rules repository:
 
 ### Contributing through Semgrep App (recommended)
 
-To contribute to the Semgrep rules repository through Semgrep App, follow these steps:
+Publish rules in the open-source Semgrep Registry to share them with the Semgrep community. To contribute and publish rules to the Semgrep Registry, follow these steps:
```

Comment:
@efortuna what more benefits can users gain from sharing their rules with the community? Potential improvements, fixes etc.? If we can mention some more, we can make this introduction a bit more rich and informative. :-)

Reply:
Hm, something like "to help more people benefit from your rule-writing efforts"? "Contribute to the field of software security"?

Reply:
Not sure here if that adds any greater value here. Should I link the private rules also below this procedure for folks who would like to avoid having their rules published so openly? @efortuna

Reply:
I said all this, but I suppose you can leave those options out. I just wanted to explain why I suggested them. It's an important part of open source culture

Reply:
@efortuna good justification, I have been convinced! Included. Please see the output now:

<img width="977" alt="Screenshot 2022-07-20 at 17 36 31" src="https://user-images.githubusercontent.com/26669307/180023983-2cdf3a84-c9af-4bb0-95fd-6e317feb58a0.png">